.github/ISSUE_TEMPLATE/custom_product/tech-day.md

```diff
@@ -49,46 +65,56 @@ This checklist is focused on a classic Jahia repository (module, app)
 - [ ] 🟠 Dependencies listed in packages.json are no more than 2 major versions behind their latest release
 - [ ] 🟠 Linting is executed properly and show no warnings
 - [ ] 🟠 No warning are presents in the browser console when using the app
+- [ ] 🙏 Remove unreachable code
```

Comment:
This is a duplicate with few [lines above](https://github.com/Jahia/.github/pull/81/files#diff-51cbdc8459eee46896ee477ef9880ed2216b827d1590baef88431b78385dc3f5R53)